### ISSUE TITLE
Multiple types for yaml.customTags

### ISSUE THREAD

**Issue by benmontour-wf** [redhat-developer/vscode-yaml#142]
Hello I'm curious if there is a way to support multiple node types for customTags? For instance in CloudFormation tags some support multiple node types. Below is a list that I attempted to use in my settings.json but this extension only seems to look at whichever it see's first and won't see the other options below. As you can see some tags, such as !GetAtt, support both sequence and scalar node types. Thoughts? Is this something that exists or is this something that still needs to be added?
```  
"yaml.customTags": [
    "!Base64 mapping",
    "!ImportValue mapping",
    "!And sequence",
    "!Equals sequence",
    "!GetAtt sequence",
    "!If sequence",
    "!FindInMap sequence",
    "!Join sequence",
    "!Not sequence",
    "!Or sequence",
    "!Select sequence",
    "!Sub sequence",
    "!Split sequence",
    "!Cidr sequence",
    "!Ref scalar",
    "!Sub scalar",
    "!GetAZs scalar",
    "!GetAtt scalar",
    "!Condition scalar",
    "!ImportValue scalar",
    "!Cidr scalar"
  ],

**Comment by JPinkney** [comment]
It would need to be added. The underlying support actually comes from https://github.com/mulesoft-labs/yaml-ast-parser so I believe the issue lays in there.

**Comment by holynakamoto** [comment]
<img width="1284" alt="screen shot 2019-02-25 at 8 15 24 am" src="https://user-images.githubusercontent.com/407440/53347066-94f84980-38d5-11e9-9f1d-628d1cf2829f.png">

**Comment by benmontour-wf** [comment]
Thank you for this fix in 0.4.0!
